### PR TITLE
Add directed attributes for filters documentation

### PR DESCRIPTION
**Pull request by llienher**
Just some more doc about filter, specifically directed attributes.

### CODE REVIEW

File: doc/integrator/filter.rst [hunk 1]
```diff
@@ -119,12 +119,19 @@ Simple example:
 Step 2: Administration interface
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The final step is to assign the enumerated attributes to the right layer in the administration.
-Go to the ``metadata`` page, add a ``enumeratedAttributes`` to an existing layer.
-The value is a single string or a list of attributes (that we defined earlier in the ``vars_<project>.yaml`` file)
-separated with a comma.
+The final step is to assign the enumerateds or directed attributes to the right layer in the administration.
```
Comment (jwkaltz) [inline doc/integrator/filter.rst]: I would say something like this, what do you think?:
"As a final step, if you wish to override the default behavior regarding how the GeoMapFish filter will provide filter attributes to the user, you can define this in the administration UI as follows."

File: doc/integrator/filter.rst [hunk 2]
```diff
@@ -119,12 +119,19 @@ Simple example:
...
+Go to the ``metadata`` page, add a ``enumeratedAttributes`` or ``directedFilterAttributes`` to an existing layer.
 
-Client-side documentation related to the enumeratedAttributes metadata is available here:
+For enumerated attributes, the value is a single string or a list of attributes (that we defined earlier
+in the ``vars_<project>.yaml`` file) separated with a comma. 
+
+For directed attributes, it is a list of attributes defined in the mapfile
+(columns and aliases from the selected table).
+
+The difference is enumerated attributes are configurable (like pointing to a specific database table),
```
Comment (jwkaltz) [inline doc/integrator/filter.rst]: "The difference is" -> "The difference is that"

File: doc/integrator/filter.rst [hunk 3]
```diff
@@ -119,12 +119,19 @@ Simple example:
...
+while directed attributes are ready-to-use values that comes directly from the mapfile configuration.
```
Comment (jwkaltz) [inline doc/integrator/filter.rst]: "that comes" -> "that come"

**Comment by eleu** [comment]
great thanks!

File: doc/integrator/filter.rst [hunk 4]
```diff
@@ -118,13 +118,22 @@ Simple example:
 
 Step 2: Administration interface
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+As a final step, if you wish to override the default behavior regarding how the GeoMapFish filter will provide filter attributes to the user,
+you can define this in the administration interface as follows.
```
Comment (sbrunner) [inline doc/integrator/filter.rst]: Cat it be reword like:

You can add some additional configuration in the administration interface as follows.

File: doc/integrator/filter.rst [hunk 5]
```diff
@@ -118,13 +118,22 @@ Simple example:
...
-The final step is to assign the enumerated attributes to the right layer in the administration.
-Go to the ``metadata`` page, add a ``enumeratedAttributes`` to an existing layer.
-The value is a single string or a list of attributes (that we defined earlier in the ``vars_<project>.yaml`` file)
-separated with a comma.
+It is possible to define enumerated or directed attributes, to wms layers only, via metadata.
+The metadata to look at is ``enumeratedAttributes`` or ``directedFilterAttributes``.
 
-Client-side documentation related to the enumeratedAttributes metadata is available here:
+For enumerated attributes, the value is a single string or a list of attributes (that we defined earlier
+in the ``vars_<project>.yaml`` file) separated with a comma.
```
Comment (sbrunner) [inline doc/integrator/filter.rst]: vars.yaml

**Pull request merged**